include/cvc5/cvc5.h

```diff
@@ -1967,10 +1970,9 @@ std::ostream& operator<<(std::ostream& out,
  * @param unordered_map The map of terms to be serialized to the given stream.
  * @return The output stream.
  */
-template <typename V>
+template <typename V> CVC5_EXPORT
```

Comment:
Is this clang-formatted? Format differs from the changes right above.

Reply:
It is not. Fixed.